### DISCUSSION TITLE
How can I get the extension install package for VS2017

### DISCUSSION THREAD

**Discussion by onWayLa** [AvaloniaUI/AvaloniaVS#483] [General]
My computer only supports the VS2017, but I want to use AvaloniaVS to develop an application. Does this extension no longer support VS2017?

**Comment by maxkatz6** [comment]
No, it does not support VS2017 anymore.

**Reply by onWayLa** [reply]
But I saw that AvaloniaVS 11.2 could support VS2017, so can I build this version code to get an extension install package for VS2017?

![image](https://github.com/AvaloniaUI/AvaloniaVS/assets/18183848/b79dc07f-870d-4ba1-98dd-b902d4f9efff)

**Reply by maxkatz6** [reply]
Yeah, you definitely can build an older extension version to get 2017 working.
